app/tray/Account/Balances/index.js

```diff
@@ -54,10 +55,12 @@ class Balances extends React.Component {
         const decimals = isNative ? nativeCurrencyInfo.decimals || 18 : rawBalance.decimals
         const symbol = (isNative && nativeCurrencyInfo.symbol) || rawBalance.symbol
 
-        return createBalance(
-          { ...rawBalance, logoURI, name, decimals, symbol },
-          ethereumNetworks[rawBalance.chainId].isTestnet ? { price: 0 } : rate.usd
-        )
+        const isTestnet = ethereumNetworks[rawBalance.chainId].isTestnet
+        const ratesExpired = Date.now() - ratesLastUpdated > RATES_EXPIRY_WINDOW
```

Comment:
how often will this re-render? we might want to think of a way to update this only so often. we can probably extract this logic out somewhere as well.